test/parallel/test-stream-pipe-objectmode-to-non-objectmode.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('node:assert');
+const { Readable, Transform, Writable } = require('node:stream');
+
+// Pipeine objects from object mode to non-object mode should throw an error and
```

Comment:
@jakecastelli sorry, just out curiosity, is pipeine a typo? should it be `pipeline`?

Reply:
Yes, it is a typo, thanks for spotting it and PR welcome 👍

Reply:
I had an other look, my origin intention should be `piping`, I've created a quick PR [here](https://github.com/nodejs/node/pull/55270#discussion_r1929865237)